gmerge

```diff
@@ -70,13 +70,13 @@ class GMerger(object):
     try:
       result = urllib2.urlopen(self.devkit_url + '/build',
                                data=self.GeneratePackageRequest(package_name))
-      print result.read()
+      sys.stdout.write(result.read())
```

Comment:
No, that causes pylint complaints about useless parentheses in Python 2.  There is a `__future__` import for a `print` function, but this seemed simpler to not add imports.